sql/sys_vars.cc

```diff
@@ -7625,6 +7626,24 @@ static Sys_var_ulonglong Sys_tf_sequence_table_max_upper_bound(
     GLOBAL_VAR(tf_sequence_table_max_upper_bound), CMD_LINE(REQUIRED_ARG),
     VALID_RANGE(1024, ULLONG_MAX), DEFAULT(1048576), BLOCK_SIZE(1));
 
+static Sys_var_charptr Sys_var_buffered_log_filename(
+    "buffered_log_filename", "Filename for buffered log",
```

Comment:
I think it we should mention error_log instead of just log

buffer_error_log_filename